src/main/java/com/cs56fitnessapp/models/Day.java

```diff
@@ -1,13 +1,33 @@
 package com.cs56fitnessapp.models;
 
 /**
- * @author Ksenia Koldaeva
- * Created: 9/17/17
- * Last Updated: 10/1/17
+ * @author Saori Shigehisa
+ * Created: 10/6/17
+ * Last Updated: 10/7/17
  */
+import java.util.ArrayList;
+import java.util.Date;
 
 public class Day implements ActivityFacts, DietFacts {
 
+    /** Date */
+    private Date date;
+
+    /** User */
+    private User user;
+
+    /** calorieGoal */
+    private double calorieGoal;
+
+    /** WorkoutList */
+    private ArrayList < Workout > workoutList;
+
+    /** MealList */
+    private ArrayList < Meal > mealList;
+
+    public boolean isToday() { return false; }
+
+    public double getCaloriesLeft() { return 0.0; }
```

Comment:
Implement getCaloriesLeft().

Reply:
Though I want to use this formula: Kcal ~= METS * bodyMassKg * timePerformingHours, there is nothing in the class, user yet, so I can't make it for now. If I make this now, I can not compile x-(